packages/sfpowerscripts-cli/messages/deploy_source.json

```diff
@@ -10,5 +10,7 @@
     "specifiedTestsFlagDescription": "Specify a comma seperated values of Apex Test that need to be executed during this deployment",
     "apexTestSuiteFlagDescription": "Name of the Apex Test Suite that needs to be executed during this deployment",
     "isToBreakBuildIfEmptyFlagDescription": "Uncheck this field, to allow for empty folders not to break build, useful in the case of pre/post step uniformity across projects",
-    "refNameFlagDescription": "Reference name to be prefixed to output variables"
+    "refNameFlagDescription": "Reference name to be prefixed to output variables",
+    "ignoreWarningsFlagDescription":"If a warning occurs and ignoreWarnings is set to true, the success field in DeployMessage is true.",
```

Comment:
Is this a bit too much implementation detail?

Reply:
Copied from sfdx-cli, happy to change

Reply:
Fixed